blueprint/api.apib

```diff
@@ -1778,6 +1922,50 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + id: 1 (string)
 + type: `comment` (string, required)
 
+## Donation Goal Attributes (object)
++ title: `Support the core team` (required, string)
++ description: `Will provide all the basic overhead costs needed to run Code Corps.` (required, string) - Several lines describing the reasons for the goal
++ amount: `1500000` (required, number) - The amount, in cents, for the goal.
++ current: `true` (required, boolean) - Indicates whether this is the current donation goal. This is determined by whether 1) this is the highest goal and the total monthly raised is higher than that amount or 2) the total monthly raised is between the lower goal's amount (if it exists) and this goal's amount.
+
+## Donation Goal Resource (object)
++ include Donation Goal Resource Identifier
++ attributes(Donation Goal Attributes)
++ relationships
+    + project
+        + data(Project Resource Identifier)
+
+## Donation Goal Resource Identifier
++ id: `1` (string, required)
++ type: `donation-goal` (string, required)
+
+## Donation Goal Response (object)
++ data(Donation Goal Resource)
++ include JSON API Version
+
+## Donation Goals Response (object)
++ data(array[Donation Goal Resource])
++ include JSON API Version
+
+## Donation Goal Create Request
++ data
+    + attributes
+        + title (string) - A title for the donation
+        + description (string) - A short plain text description for the donation
+        + amount (number) - the amount of money in cents for the goal
+        + current(boolean) - indicates current donation goal
+    + relationships
+        + project
+            + data(Project Resource Identifier) - Identifier of project this donation goal is being created for
```

Comment:
This object is useful but it looks like it's not being used.

I don't think we'd be setting `current`, though, manually when creating this goal.